bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -302,8 +302,8 @@ metadata:
           env:
           - name: INSTALL_SCOPE
             value: namespaced
-    olm.skipRange: '>=3.3.0 <3.7.1'
-    operators.operatorframework.io/builder: operator-sdk-v1.2.0
+    olm.skipRange: ">=3.5.0 <3.7.1"
```

Comment:
Any reason we need to change it to >=3.5.0?

Reply:
Is it because https://github.com/IBM/ibm-common-service-operator/blob/7e263de5c0bfbd5eeb233373ca8a523afcf6c6c6/Makefile#L208

Reply:
no, that is a typo. Good catch